src/plugins/custom_integrations/README.md

```diff
@@ -0,0 +1,9 @@
+# customIntegrations
```

Comment:
The docs will need fleshing out and conversion to `.mdx` if we want to include them in the new dev docs

Reply:
thx.  my preference is to hold off on dev-docs in this PR. These will evolve fairly quickly. Would prefer to introduce only in separate PR when contract stabilizes.